R/registry.R

```diff
@@ -9,7 +9,7 @@
 #'
 #' @format An environment.
 #' @keywords internal
-gargle_env <- new.env()
+gargle_env <- new.env(parent = emptyenv())
```

Comment:
Yeah. [To quote](http://adv-r.had.co.nz/Environments.html#explicit-envs) @hadley:

> Just as you can use a list to pass data between functions, you can also use an environment. When creating your own environment, note that you should set its parent environment to be the empty environment. This ensures you don’t accidentally inherit objects from somewhere else.